.orchestra/config/components/revng_container.yml

```diff
@@ -0,0 +1,26 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def revng_container():
+binary_archives: private
+configure: |
+  mkdir -p "$BUILD_DIR" && cd "$BUILD_DIR"
+  touch empty_directory_ignore_it
+install: |
+  INSTALLER="$ORCHESTRA_DOTDIR/support/revng-container/install.sh"
+  #echo "bacb4190d2457e2cc2303aed9928f2a3cf33107fe20c0613b68fedf56683f4f9 $INSTALLER" | \
+  #  sha256sum --quiet -c -
```

Comment:
?

Reply:
Must have forgot to uncomment and update the hash, fixed